### PR TITLE
API设计

### PR DESCRIPTION
**Pull request by Ccrool**

### CODE REVIEW

**Comment by Ccrool** [comment]
修改

File: hja/2017-11-29/API文档.txt [hunk 1]
```diff
@@ -0,0 +1,97 @@
+﻿WEB URL
+
+
+HTML页面
+
+
+1.首页：/index
```
Comment (calidion) [inline hja/2017-11-29/API文档.txt]: /index
=>
/

File: hja/2017-11-29/API文档.txt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+
+6.邮件总览：/mails/read：
+
+
+  作用：用户浏览所有收到的邮件列表
+
+
+
+7.邮件详情：/mails/showdetail：
```
Comment (calidion) [inline hja/2017-11-29/API文档.txt]: /mails/showdetail：
=>
/mails/detail/

File: hja/2017-11-29/API文档.txt [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+action等于login时，进行登录操作
+	
+
+        data:action等于register时，进行注册操作
+
+
+	data:action等于namecheck时，进行用户名是否占用查询操作
+
+
+	data:action等于receivercheck时，进行收件人是否存在的查询操作
```
Comment (calidion) [inline hja/2017-11-29/API文档.txt]: 考虑 
GET /users?state=exist&username=eric&id=1

**Comment by Ccrool** [comment]
修改

**Pull request merged**